src/lib.rs

```diff
@@ -234,7 +234,7 @@ trait GlObject {
     fn get_id(&self) -> gl::types::GLuint;
 }
 
-/// Internal trait for enums that can be turned into GLenum.
+/// Internal trait for enums that can be turned into GLEnum.
```

Comment:
Ah OK, I changed it because of the trait with capital E, although that also has a lower-case L. I'll put it back :)